src/test/features/response/routes/view-reponse.ts

```diff
@@ -0,0 +1,58 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+
+import { attachDefaultHooks } from '../../../routes/hooks'
+import { checkAuthorizationGuards } from './checks/authorization-check'
+
+import { Paths as ResponsePaths } from 'response/paths'
+
+import { app } from '../../../../main/app'
+
+import * as idamServiceMock from '../../../http-mocks/idam'
+import * as claimStoreServiceMock from '../../../http-mocks/claim-store'
+
+import { checkCountyCourtJudgmentRequestedGuard } from './checks/ccj-requested-check'
+import { checkNotDefendantInCaseGuard } from './checks/not-defendant-in-case-check'
+import * as draftStoreServiceMock from '../../../http-mocks/draft-store'
+
+const cookieName: string = config.get<string>('session.cookieName')
+const pagePath: string = ResponsePaths.viewResponsePage.evaluateUri({ externalId: claimStoreServiceMock.sampleClaimObj.externalId })
+
+describe('Defendant response: confirmation page', () => {
+  attachDefaultHooks(app)
+
+  describe('on GET', () => {
+    const method = 'get'
+    checkAuthorizationGuards(app, method, pagePath)
+    checkNotDefendantInCaseGuard(app, method, pagePath)
+
+    describe('for authorized user', () => {
+      beforeEach(() => {
+        idamServiceMock.resolveRetrieveUserFor(claimStoreServiceMock.sampleClaimObj.defendantId, 'citizen')
+      })
+
+      checkCountyCourtJudgmentRequestedGuard(app, method, pagePath)
+
+      it('should render page when everything is fine', async () => {
```

Comment:
One test case needed to check the behaviour if response has not been submitted yet.